### PR TITLE
Create db measurement functions

### LINKED ISSUE #91: Create DB function station_data.create_measurement_view(...)
Create DB function `station_data.create_measurement_view(station_id, pollutant)`

### LINKED ISSUE #92: Add measurement layer creation for station_layer_creator
Add `"measurement"` layer creation to `station_layer_creator` service.

Convention: If we detect a "prediction" layer we have to create a `"measurement"`.

Precondition: DB function to cerate measurment view: `station_data.create_prediction_view(...)` -> #91

### PR DESCRIPTION
**Pull request by jdk24**
Add functions in db init to create views for measurements similar to prediction views.

Closes #91 

Function: * **station_data.create_measurement_view**(**station_code** text, **component_name** text)*

### CODE REVIEW

File: db/update-files.d/9_create_measurement_views.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+				json_agg(json_build_object(''datetime'', sel.date_time, ''val'', sel.wert) ORDER BY sel.date_time DESC)::text AS series,
+				s.wkb_geometry AS geom
+				FROM sel
+				JOIN station_data.lut_component co ON sel.fk_component = co.idpk_component
+				JOIN station_data.lut_station s ON sel.fk_station = s.idpk_station
+			 GROUP BY s.idpk_station, sel.fk_component, sel.fk_station, co.component_name, s.station_name, s.wkb_geometry;
+			 
+			 GRANT SELECT ON station_data.%1$s_measurement_%2$s TO app;
+			 
+			', replace(lower(station_code),'-','_'), lower(component_name), station_code, component_name);
```
Comment (JakobMiksch) [inline db/update-files.d/9_create_measurement_views.sql:49]: a comment might be helpful why `replace(lower(station_code),'-','_')` is necessary

**Review by JakobMiksch — APPROVED** [review]
Looks good to me. 
A general description what the function does would be very helpful

**Review by chrismayer — APPROVED** [review]
LGTM, thanks.

**Comment by jdk24** [comment]
Documentation added as mentioned by @JakobMiksch, @chrismayer thank you for the review.

**Pull request merged**